lib/sync/stop-updating-store.js

```diff
@@ -0,0 +1,8 @@
+const storageManager = require('brainstem-js').StorageManager.get();
+const getStorageManagerListener = require('./storage-manager-listener');
+
+module.exports = () => {
+  for (const collectionName of storageManager.collectionNames()) {
```

Comment:
Food for thought: Creating a helper method to encapsulating pulling in the storage manager and this for loop. Effectively we'd be making our own `storageManagerIterator` which does a each over the collections and accepts a callback it would run on each collection.

Might be a nice way to give us a helper method we could use to DRY these loops up and for use in the future.